python/graph_coloring.py

```diff
@@ -43,9 +43,9 @@ def _run_algorithm(graph: Graph, parameters) -> List[int]:
         "no_of_processes": 1,
         "no_of_chunks": 1,
         "communication_delay": 10,
-        "max_iterations": max_iterations,
+        "max_iterations": 10,
```

Comment:
These are default parameters, where are input parameters used in this chunk of code?